statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -159,6 +164,11 @@ def fit(self, start_params=None, transformed=True,
         return_params : boolean, optional
             Whether or not to return only the array of maximizing parameters.
             Default is False.
+        hessian_method : {'opg','oim','cs'}, optional
+            The method by which the Hessian is numerically approximated. 'opg'
+            uses outer product of gradients, 'oim' uses the information
+            matrix formula from Harvey (1989), and 'cs' uses second-order
```

Comment:
oim Harvey is an approximation, is it?
cs is real hessian.

If I remember this correctly, then it might be better to use `oim` for cs, and `oim_harvey` or something like this for current oim